feat: rewrite js rpg project (#46515)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3b365f1cdeb33efc2502e.md

---
id: 62a3b365f1cdeb33efc2502e
title: Step 22
challengeType: 0
dashedName: step-22
---

# --description--

> You have been declaring your variables with the `var` keyword. However, in modern JavaScript it is best practice to use the `let` keyword instead. This fixes a number of unusual behaviors with `var` that can make your code difficult to debug.

Reviewer: ```suggestion
You have been declaring your variables with the `var` keyword. However, in modern JavaScript it is best practice to use the `let` keyword instead. This fixes several unusual behaviors with `var` that can make your code difficult to debug.
```
